rocky/assets/js/toggleSwitch.js

```diff
@@ -0,0 +1,51 @@
+import { onDomReady } from "../js/imports/utils.js";
+
+onDomReady(initToggleSwitches);
+
+function initToggleSwitches() {
+  // Switches get stitches ;-)
+  let toggle_switches = document.querySelectorAll(".toggle-switch");
+
+  for (let i = 0; i < toggle_switches.length; i++) {
+    let options = toggle_switches[i].querySelectorAll(".toggle-switch-button");
+
+    for (let j = 0; j < options.length; j++) {
+      let option = options[j];
+
+      // Hide all elements linked to toggle switch options.
+      document
+        .getElementById(option.getAttribute("data-target-id"))
+        .classList.add("hidden");
```

Comment:
I'd say, Hide all of them by applyting a className to it.
And then use the :target selector and the url to show the one being active. that moves the state-management from JS to Css for most of this.
It would not be compatible with nested versions of this component, as selecting a nested 'tab' would hide the parent tab, But I don't think we have that usecase anyway.

Reply:
While it would be nice if state management didn't require JS, it's unfortunately not possible to make this fully accessible without using JS to manage ARIA attributes.

@underdarknl could you please verify whether the APG's [Tabs pattern](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/) aligns with the expected use here, as I suspect? If so I would implore you to follow one of the APG's example implementations. If not, let's look together for fitting ARIA roles and behaviours.

Reply:
I do think the Tabs pattern fits this nicely. Do you have a preference on which example to follow? Especially if we want to move this to Manon upstream at some point?

Reply:
I don't know the exact context. If switching accidentally would be a problem, manual activation would make sense. For upstream we could add both and make it configurable.